### ISSUE TITLE
Unclear error message for unauthorized user

### ISSUE THREAD

**Issue by joel-hamill** [mesosphere/dcos-cli#455]
When an unauthorized user tries to install a service in the CLI they are presented with this message:
```
$ dcos package install spark
Note that the Apache Spark DCOS Service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.
We recommend a minimum of two nodes with at least 2 CPU and 2GB of RAM available for the Spark Service and running a Spark job.
Note: The Spark CLI may take up to 5min to download depending on your connection.
Continue installing? [yes/no] yes
Installing Marathon app for package [spark] version [1.6.0]
54.149.70.108's username: alic
alic@54.149.70.108's password: 
54.149.70.108's username: alice
alice@54.149.70.108's password: 
Error decoding response from [http://54.149.70.108/marathon/v2/info]: HTTP 403: Forbidden
```

This error should be made clearer - "Error decoding response" is ambiguous.

**Comment by tamarrow** [comment]
What auth type are you using? Any wording recommendations?

**Comment by joel-hamill** [comment]
@tamarrow i'm using the default auth with 1.6. i think RBAC?
[2:52 PM] 
here's a working draft of th topic: https://docs.mesosphere.com/installing-enterprise-edition-1-6/enabling-authorization/.

How about rewording to `You are not authorized to perform this operation.`

**Comment by joel-hamill** [comment]
For Kafka, I get a different message: ```$ dcos kafka broker list
54.149.70.108's username: joel
joel@54.149.70.108's password: 
Error: java.io.IOException: 401 - Unauthorized```

Are these messages service-specific?

**Comment by tamarrow** [comment]
The kafka error is in the kafka subcommand, which is up to whoever owns the kafka cli. I'll look into the other one.

**Comment by joel-hamill** [comment]
@tamarrow Okay, I have opened a JIRA to track the Kafka issue: https://mesosphere.atlassian.net/browse/INFINITY-68.
